src/emailmessage.cpp

```diff
@@ -1164,14 +1161,7 @@ void EmailMessage::buildMessage(QMailMessage *msg)
         type.setSubType("html");
     */
     // This should be improved to use QuotedPrintable when appending parts and inline references are implemented
-    if (m_attachments.size() == 0) {
-        msg->setBody(QMailMessageBody::fromData(m_bodyText, type, QMailMessageBody::Base64));
-    } else {
-        QMailMessagePart body;
-        body.setBody(QMailMessageBody::fromData(m_bodyText.toUtf8(), type, QMailMessageBody::Base64));
```

Comment:
Ah, there is this inconsistency in the original code also that use the QString m_bodyText in one part of the if, and use its QByteArray representation in UTF8 in the other branch (which will be automatically converted back into a QString as far as I know).

Reply:
Guess so...